src/care/host_device_ptr.h

```diff
@@ -422,6 +438,17 @@ namespace care {
    void using_host_device_ptr_outside_of_raja_loop_not_allowed(T foo); 
 } // namespace care
 
+#if !defined(CARE_ENABLE_IMPLICIT_CONVERSIONS)
+
+// TODO: Declaring these functions causes problems with a project that depends on CARE
```

Comment:
Thanks for figuring this out

Reply:
No problem. I've got some ideas for a better solution, but they should be in a separate pull request.